static_report/src/components/IndexReport/IndexReport.tsx

```diff
@@ -0,0 +1,268 @@
+import {
+  Box,
+  Button,
+  Center,
+  Checkbox,
+  Flex,
+  HStack,
+  Heading,
+  LinkBox,
+  LinkOverlay,
+  Tab,
+  TabList,
+  TabPanel,
+  TabPanels,
+  Table,
+  TableContainer,
+  Tabs,
+  Tbody,
+  Td,
+  Text,
+  Th,
+  Thead,
+  Tr,
+  VStack,
+} from '@chakra-ui/react';
+
+import { useState } from 'react';
+import { Main } from '../shared/Main';
+import { formatReportTime } from '../../utils/formatters';
+
+function StickyFooter({
+  checkState,
+  onCompareClick,
+  error,
+}: {
+  checkState: string[];
+  onCompareClick: () => void;
+  error?: string;
+}) {
+  return (
+    <Center>
+      <Box
+        bgColor="gray.100"
+        borderRadius="15px"
+        bottom="10px"
+        opacity="0.95"
+        p={4}
+        pos="fixed"
+        w="40%"
+        zIndex={3}
+      >
+        <Center>
+          <VStack>
+            <HStack spacing="20px">
+              <Box w="56px" textAlign={['left']}>
+                <Text fontWeight="semibold">Base: </Text>
+                <Text fontWeight="semibold">Target: </Text>
+              </Box>
+              <Box>
+                <Text>{checkState[0]}</Text>
+                <Text>{checkState[1]}</Text>
+              </Box>
+              <Box m={'auto 0'}>
+                <Button colorScheme="blue" onClick={onCompareClick}>
+                  Compare
+                </Button>
+              </Box>
+            </HStack>
+            {error && (
+              <Text color="red.500" fontWeight="semibold">
+                {error}
+              </Text>
+            )}
+          </VStack>
+        </Center>
+      </Box>
+    </Center>
+  );
+}
+
+export function IndexReport({ data }) {
```

Comment:
You probably don't need this `data` prop at the root app shell level. 
You can directly read `window.<>` in the`*Report`, all other Route page Component can then have the `data` prop signature.

Responsibility for data should be page-level, in which all UI components consume them

E.g. ![image](https://user-images.githubusercontent.com/11186480/184756233-c9360f00-1c91-40a3-b79d-716eabc3573a.png)

Reply:
Do you mean removing `data` prop in the app shell level and handling `data` only in IndexReport?

Reply:
yes 👍
As in only components downstream inside of `IndexReport` should have the data prop.